src/components/TaskCard/AddTaskCard.jsx

```diff
@@ -57,7 +57,17 @@ const AddTaskCard = ({ showCreateCard, setShowCreateCard }) => {
         className={style.textarea}
       ></textarea>
       <small className={style.error}>{error && error}</small>
-      <button onClick={storeTask}>Add Task</button>
+
+      <div className={style.actionButtonContainer}>
+        <button className={style.button} onClick={storeTask}>
+          Add Task
+        </button>
+        <img
+          src={DELETE}
```

Comment:
constant without a context, mis-represents an action.

```suggestion
          src={ICON_DELETE}
```